data_processors/pipeline/orchestration/dragen_wts_step.py

```diff
@@ -26,90 +26,99 @@
 def perform(this_workflow: Workflow):
     logger.info(f"Preparing {WorkflowType.DRAGEN_WTS.value} workflows")
 
-    batcher = Batcher(
-        workflow=this_workflow,
-        run_step=WorkflowType.DRAGEN_WTS.value,
-        batch_srv=batch_srv,
-        fastq_srv=fastq_srv,
-        logger=logger,
+    this_sqr = this_workflow.sequence_run
+
+    # Check if all other DRAGEN_WTS_QC workflows for this run have finished
+    # If yes we continue to the DRAGEN WTS Workflow
+    # If not we wait (until all DRAGEN_WTS_QC worklfows for this run have finished
```

Comment:
typo in comment

```
workflows
```

Reply:
Resolved in 7f4f03c68c8146c4034f141bf454a5ff7b4f5321